app/helpers/hyku_knapsack/application_helper.rb

```diff
@@ -15,25 +15,6 @@ def video_embed_viewer_display(work_presenter, locals = {})
     def video_embed_viewer_display_partial(work_presenter)
       'hyrax/base/' + work_presenter.video_embed_viewer.to_s
     end
-
-    ##
-    # This is in place to coerce the :q string to :query for passing the :q value to the query value
-    # of a IIIF Print manifest.
-    #
-    # @param doc [SolrDocument]
-    # @param request [ActionDispatch::Request]
-    def generate_work_url(doc, request)
-      url = super
-      return url if request.params[:q].blank?
-
-      key = doc.any_highlighting? ? 'parent_query' : 'query'
```

Comment:
Do we know what the diff of sending parent_query and query is? I don't think Hyku sends query

Reply:
That's why I deleted this... we refactored hyku and this was no longer applicable.